Misc fixes for virtualenv / pip

undingen opened the pull request:
Hopefully one of the last sets of patches required.
- Update libpypa to fix another missing newline at end of file bug.
- Set distutils package install directory
- zipimporter: Ignore bytecode files inside archives (This let's them still load if the souce file can get found)

kmod commented on from_cpython/Lib/distutils/command/install.py:
```diff
@@ -41,8 +41,8 @@
-        'purelib': '$base/lib/python$py_version_short/site-packages',
-        'platlib': '$platbase/lib/python$py_version_short/site-packages',
+        'purelib': '$base/site-packages',
+        'platlib': '$base/site-packages',
```
can we add a "Pyston change" comment here?

undingen commented:
- I added the ```Pyston changed``` comment
- Cleaned up yesterdays commit #408 
- Changed a error message slightly to output same string as my ubuntu cpython 2.7.8

kmod commented:
cool thanks :)
